files/templates/additional-scrape-configs.template.yaml

```diff
@@ -243,3 +243,36 @@
   # Add provider label.
   - target_label: provider
     replacement: {{ .Provider }}
+# kube-controller-manager
+- job_name: {{ .ClusterID }}-prometheus/kubernetes-controller-manager-{{ .ClusterID }}/0
+  honor_timestamps: false
+  metrics_path: /metrics
+  scheme: https
+  kubernetes_sd_configs:
+  - api_server: https://{{ .APIServerURL }}
+    role: pod
+    tls_config:
+      ca_file: /etc/prometheus/secrets/{{ .SecretName }}/ca
+      cert_file: /etc/prometheus/secrets/{{ .SecretName }}/crt
+      key_file: /etc/prometheus/secrets/{{ .SecretName }}/key
+      insecure_skip_verify: false
+    namespaces:
+      names:
+      - kube-system
+  tls_config:
+    ca_file: /etc/prometheus/secrets/{{ .SecretName }}/ca
+    cert_file: /etc/prometheus/secrets/{{ .SecretName }}/crt
+    key_file: /etc/prometheus/secrets/{{ .SecretName }}/key
+    insecure_skip_verify: true
+  relabel_configs:
+  - source_labels: [__meta_kubernetes_pod_label_k8s_app]
+    regex: controller-manager
+    action: keep
+  - source_labels: [__meta_kubernetes_pod_name]
```

Comment:
Why do we rename the pod name to app? Should the app name not be kubernetes-controller-manager or controller-manager instead?

Reply:
something like this right?

```yaml
  - target_label: app
    replacement: kube-controller-manager
```

Reply:
Yep exactly

Reply:
done